rocketmq-streams-commons/src/main/java/org/apache/rocketmq/streams/common/context/UserDefinedMessage.java

```diff
@@ -318,12 +318,12 @@ public Date getDate(String key) {
 
     @Override
     public java.sql.Date getSqlDate(String key) {
-        return super.getSqlDate(key);
+        return (java.sql.Date) super.getSqlDate(key);
```

Comment:
It is cause by update fastjson version, and I fixed it by changed the return type.